backend/app/views/spree/admin/users/_form.html.erb

```diff
@@ -60,10 +60,13 @@
       <%= f.error_message_on :password %>
     <% end %>
 
-    <%= f.field_container :password do %>
-      <%= f.label :password_confirmation %>
-      <%= f.password_field :password_confirmation, :class => 'fullwidth' %>
-      <%= f.error_message_on :password_confirmation %>
+
+    <% if f.object.respond_to?(:password_confirmation) %>
```

Comment:
Not a big deal, but just for consistency with the [related factory PR](https://github.com/solidusio/solidus/pull/1445) should we use `f.object.class.attribute_method?(:password_confirmation)`